distributed/client.py

```diff
@@ -107,9 +117,14 @@ def done(self):
         """ Is the computation complete? """
         return self.event.is_set()
 
-    def result(self):
-        """ Wait until computation completes. Gather result to local process """
-        result = sync(self.client.loop, self._result, raiseit=False)
+    def result(self, timeout=None):
+        """ Wait until computation completes. Gather result to local process.
+
+        If `timeout` seconds are elapsed before returning, a TimeoutError
```

Comment:
I think that sphinx uses restructured text by default, so you'll need double backticks if you want to use code highlighting.